lib/data_update_scripts/20210305222641_resave_users_for_imgproxy_update.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class ResaveUsersForImgproxyUpdate
+    def run
+      return if SiteConfig.dev_to?
```

Comment:
Are we not worried about it for DEV or do we have a different process for it due to scale?

Reply:
I want to also point out that I'm resaving Users to invalidate their profile page's profile image cache.